js/src/admin/components/PermissionGrid.js

```diff
@@ -158,12 +163,18 @@ export default class PermissionGrid extends Component {
       permission: 'user.viewLastSeenAt',
     });
 
+    items.merge(this.getExtensionPermissions('view'));
+
     return items;
   }
 
   startItems() {
     const items = new ItemList();
 
+    if (this.extensionId) {
```

Comment:
Is there anyway all this logic could be moved into ExtensionPermissionGrid? I don't really like PermissionGrid itself being aware of a current extension